airsenal/framework/optimization_utils.py

```diff
@@ -147,6 +147,10 @@ def get_discount_factor(next_gw, pred_gw, discount_type="exp", discount=14 / 15)
     if discount_type not in allowed_types:
         raise Exception("unrecognised discount type, should be exp or const")
 
+    if not next_gw:
+        # during tests 'none' is passed as the root gw, default to zero so the
+        # optimisation is done solely on pred_gw ahead.
+        next_gw = 0
```

Comment:
Would `next_gw = 1` make more sense here? Or maybe `next_gw = pred_gw` so that the default is to not apply a discount, effectively?